outputs.tf

```
@@ -19,6 +19,11 @@ output "topic" {
   description = "The name of the Pub/Sub topic"
 }
 
+output "topic_link" {
```

Let's stick with the term "ID".

```suggestion
output "id" {
```